input/resources/ValueSet-ehealth-program.json

```diff
@@ -0,0 +1,31 @@
+{
+  "resourceType": "ValueSet",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/vs/ehealth-program",
```

Comment:
Burde det være plural? "programs".

Reply:
Either will work for the ValueSet, suggest singular.